### PR TITLE
[Australia]- set up landing page and site

### PR DESCRIPTION
**Pull request by swapnilogale**
----
📚 Documentation preview 📚: https://writethedocs-www--2140.org.readthedocs.build/

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs can you approve the landing page so that we can announce it in the May newsletter?

File: docs/conf/australia/2024/news/welcome.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
+:template: {{year}}/generic.html
+
+.. post:: May 10, 2024
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/welcome.rst]: Wanna tweak this to 6th? So it shows on the landing page.

File: docs/conf/australia/2024/news/welcome.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Announcing Write the Docs Australia {{year}}
+==============================================
+
+G'day and welcome to the official launch of `Write the Docs Australia {{year}} <https://www.writethedocs.org/conf/australia/{{year}}/>`_,
+our seventh conference.
+
+We're excited to announce the dates for our next conference:
+**Write the Docs Australia 2024 will be on November 28-29, in Melbourne**.
+
+Write the Docs Australia will be an in-person conference, similar to our pre-COVID conferences.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/welcome.rst]: Last year was in person too right?

File: docs/conf/australia/2024/news/welcome.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+.. _Library at the Dock: https://www.melbourne.vic.gov.au/community/hubs-bookable-spaces/the-dock/library-at-the-dock/Pages/library-at-the-dock.aspx
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We plan to launch our CFP early to mid June and have it last until sometime until mid-September.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/welcome.rst]:
```suggestion
We plan to launch our CFP early to mid June and have it last until mid-September.
```

File: docs/conf/australia/2024/news/welcome.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Get started coming up with ideas for the conference now.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/welcome.rst:45]:
```suggestion
Start thinking about  ideas for your talk now. ;-)
```

File: docs/conf/australia/2024/news/welcome.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+As always, we keep ticket prices low so that it is accessible to the widest range of people possible.
+However, if you can't afford these prices and still wish to attend, we will bringing our Opportunity Grants program back again this year.
+
+Sponsorship
+-----------
+
+We are planning to do similar sponsorship levels again this year as last year.
+In addition to the regular sponsorship levels, we have additional ways to sponsor our online events.
+See our `sponsorship prospectus <https://www.writethedocs.org/conf/australia/2023/sponsors/prospectus/>`_ for all the options.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/welcome.rst]:
```suggestion
See our `sponsorship prospectus <https://www.writethedocs.org/conf/australia/2024/sponsors/prospectus/>`_ for all the options.
```

Comment (plaindocs) [reply docs/conf/australia/2024/news/welcome.rst]: Is the prospectus updated? @swapnilogale

**Review by plaindocs — COMMENTED** [review]
OK, I've looked at the Welcome post and the landing page *only* which look ok at this point. 
 
- [x] Sponsorship page update

File: docs/conf/australia/2024/news/welcome.rst (hunk 5 above)
Comment (swapnilogale) [reply docs/conf/australia/2024/news/welcome.rst]: It is now!

File: docs/_data/australia-2024-config.yaml [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    event: Main Conference
+    date: November 28-29
+    summary: The main days of the conference. We will be running one main track, and an <a href="/conf/australia/2024/unconference/">Unconference</a>.
+    icon: conference
+
+#    summary: The main days of the conference. We will be running one main track, and an Unconference.
+  # These are listed in the schedule
+  day_one:
+    event: Outing
+    date: On your own or community-sourced
```
Comment (plaindocs) [inline docs/_data/australia-2024-config.yaml]:
```suggestion
    date: Self-organized
```

**Comment by swapnilogale** [comment]
@plaindocs can you review and approve? Venue has been finalised and signed off.

File: docs/conf/australia/2024/news/welcome.rst [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+As always, we keep ticket prices low so that it is accessible to the widest range of people possible.
+However, if you can't afford these prices and still wish to attend, we will bringing our Opportunity Grants program back again this year.
+
+Sponsorship
+-----------
+
+We are planning to do similar sponsorship levels again this year as last year.
+In addition to the regular sponsorship levels, we have additional ways to sponsor our online events.
+See our `sponsorship prospectus <https://www.writethedocs.org/conf/australia/2024/sponsors/prospectus/>`_ for all the options.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/welcome.rst:61]: Prospectus is linked from, here :-D

**Review by plaindocs — APPROVED** [review]
Looks OK. Venue, date, welcome post. Prospectus is linked through the post.

**Pull request merged**